plugins/modules/rulebook_activation.py

```diff
@@ -164,13 +172,62 @@
     project_name: "Example Project"
     rulebook_name: "hello_controller.yml"
     decision_environment_name: "Example Decision Environment"
-    enabled: false
-    awx_token_name: "Example Token"
     organization_name: "Default"
+    eda_credentials:
+        - My AAP Credentials
     event_streams:
       - event_stream: "Example Event Stream"
         source_name: "Sample source"
 
+- name: Update rulebook activation fields
```

Comment:
We need another example to create an activation disabled, for the new interface.